pnpm/test/monorepo/dedupePeers.test.ts

```diff
@@ -91,6 +91,59 @@ auto-install-peers=false`, 'utf8')
   expect(loadJsonFile<any>('project-2/package.json').dependencies['@pnpm.e2e/abc-grand-parent-with-c']).toBe('^1.0.1') // eslint-disable-line
 })
 
+// Covers https://github.com/pnpm/pnpm/issues/8877
+test('partial update --latest in a workspace should not affect other packages when dedupe-peer-dependents is true', async () => {
```

Comment:
This is essentially a duplicate of this other test that I added: https://github.com/pnpm/pnpm/pull/8905/files#diff-c716835252557336d8471f4ef5956b804e48e1c061cb9c78806127e37a42dd3eR450, just a tad more end-to-end. Since `dedupePeers` is tested here, it felt appropriate to include it, but I'm happy to remove either one of the tests, if you think it is best 😌

Reply:
if there are two tests that are exactly the same, then remove one of them.

Reply:
Yeah, I agree! I was wondering if you had a preference. I ended up keeping the `dedupePeers` test in the `pnpm` package in 48884797dbbd6001c53e9007c707a10234cd4a30, so the test can live alongside the other tests for this functionality